### PR TITLE
NXP-27652: refactor nuxeo-styles

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: themes/base.js [hunk 1]
```diff
@@ -556,6 +576,20 @@ const template = html`
         --nuxeo-suggester-media-width: calc(100% - 90px);
 
         --nuxeo-suggester-media-margin-left: 1.2rem;
+
+        --nuxeo-document-content-min-height: calc(100vh - 216px - var(--nuxeo-app-top));
```
Comment (nelsonsilva) [inline themes/base.js]: I'd do away with all the margin-bottom vars (and document content and trash content heights too if possible)

Comment (Gabez0r) [reply themes/base.js]: We're already using it on [nuxeo-document-page](https://github.com/nuxeo/nuxeo-web-ui/blob/master/elements/document/nuxeo-document-page.js#L127). We can delete `--nuxeo-results-view-margin-bottom`, though.

**Comment by guirenard** [comment]
We have more layouts to update in https://github.com/nuxeo/nuxeo-web-ui/tree/master/plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document

**Comment by Gabez0r** [comment]
✅ https://qa.nuxeo.org/jenkins/view/All/job/web-ui-pipeline-gbarata/242/

File: plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-create-layout.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 -->
 <dom-module id="nuxeo-multicomplex-create-layout">
   <template>
-    <style>
+    <style include="nuxeo-styles">
       *[role='widget'] {
```
Comment (richardsd) [inline plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-create-layout.html]: Shouldn't we remove the ***[role='widget']** rule here too? Or am I missing something?

File: plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-edit-layout.html [hunk 3]
```diff
@@ -5,11 +5,7 @@
 -->
 <dom-module id="nuxeo-multicomplex-edit-layout">
   <template>
-    <style>
```
Comment (richardsd) [inline plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-edit-layout.html]: Just like you did here 😄

File: plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-create-layout.html (hunk 2 above)
Comment (richardsd) [reply plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-create-layout.html]: or change to ?
```
:host {
  @apply --paper-card;
}
```

Comment (Gabez0r) [reply plugin/itests/addon/src/main/resources/web/nuxeo.war/ui/document/multicomplex/nuxeo-multicomplex-create-layout.html]: Yes, we should remove the `*[role='widget'] `. Thank you. No need to add the paper card here. We usually only use it on view layouts.

**Pull request merged**